update-orphaning/index.html

```diff
@@ -1,206 +1,1086 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Update orphaning dashboard</title>
+<title>Application Update Out Of Date Dashboard</title>
 <meta charset="utf-8">
-<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"> </script>
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script>
 <script type="text/javascript" src="https://d3js.org/d3.v3.min.js" charset="utf-8"></script>
 <script type="text/javascript" src="https://telemetry.mozilla.org/new-pipeline/lib/d3pie.min.js"></script>
 <style>
-
 body {
   font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+#mainDiv {
   position: relative;
   width: 960px;
   margin: 1em auto 4em;
 }
-.update-disabled-chart div {
-  font: 10px sans-serif;
-  background-color: steelblue;
-  text-align: right;
-  padding: 3px;
-  margin: 1px;
-  color: white;
+#weeklyDropdownDiv {
+  border-radius: 5px;
+  border: 2px solid #aaa;
+  background-color: #f0f0f2;
+  padding: 7px;
+  z-index: 10;
+  position: fixed;
 }
-.update-check-code-notify-chart div {
-  font: 10px sans-serif;
-  background-color: steelblue;
-  text-align: right;
-  padding: 3px;
-  margin: 1px;
-  color: white;
+#weeklyDropdown {
+  text-align: center;
+  width: 100%;
 }
-
-#update-ping-code-description {
-  height: 50px;
+.p0_title,
+.p1_title,
+.p2_title,
+.p3_title {
```

Comment:
These are d3pie generated elements that don't have a method to set the class when they are generated.